src/app/api/[...path]/route.ts

```diff
@@ -0,0 +1,47 @@
+import { NextRequest, NextResponse } from "next/server";
+
+async function handleRequest(req: NextRequest, params: { path: string[] }, method: "GET" | "POST") {
+  try {
+    const API_HOST = process.env.API_HOST;
+    if (!API_HOST) {
+      throw new Error("API_HOST is not defined");
+    }
+
+    const endpoint = params.path.join("/");
+    const fullUrl = `${API_HOST}/api/v1/${endpoint}`;
+
+    if (method === "POST") {
```

Comment:
`req.method` 를 사용하지 않고 `method` param 을 하나 더 받는 이유가 있을까 ?

```suggestion
    if (req.method === "POST")
```

Reply:
get이랑 post만 구현해놔서 명시하도록 했는데, 상관없을것같당!! 반영했어!